doc/src/build/programming-with-objects/ch1-object-basics.md

```diff
@@ -71,30 +71,34 @@ public fun create(red: u8, green: u8, blue: u8, ctx: &mut TxContext) {
 ```
 > :bulb: Naming convention: Constructors are typically named **`new`**, which returns an instance of the struct type. The **`create`** function is typically defined as an entry function that constructs the struct and transfers it to the desired owner (most commonly the sender).
 
-We can also add a getter to `ColorObject` that returns the color values, so that modules outside of `ColorObject` are able to read their values:
+We can also add a getter to `ColorObject` that returns the color values so that modules outside of `ColorObject` are able to read their values:
 ```rust
 public fun get_color(self: &ColorObject): (u8, u8, u8) {
     (self.red, self.green, self.blue)
 }
 ```
 
-You can find the full code [ColorObject.move](../../../move_code/objects_tutorial/sources/Ch1_2/ColorObject.move).
+Find the full code online in [ColorObject.move](../../../move_code/objects_tutorial/sources/Ch1_2/ColorObject.move).
 
-To compile the code, make sure you have followed the [Sui installation guide](../install.md) so that `sui-move` is in `PATH`. In the code [root directory](../../../move_code/objects_tutorial/) (where `Move.toml` is), you can run:
+To compile the code, make sure you have [installed Sui](../install.md) so that `sui-move` is in `PATH`. In the code [root directory](../../../move_code/objects_tutorial/) (where `Move.toml` is), run:
 ```
 sui-move build
 ```
 
-### Writing Unit Tests
-After defining the `create` function, we want to test this function in Move using unit tests, without having to go all the way to sending Sui transactions. Since Sui manages global storage separately outside of Move, there is no direct way to retrieve objects from global storage within Move. This poses a question: after calling the `create` function, how do we check that the object is properly transferred?
+TODO: Doesn't this assume they have downloaded the entire `sui` repo too? `install.md` gets them the `sui-move` binary but not the repo. Adam asked me to keep that out of install and so instead lives at [Contribute to Sui](../../contribute/index.md) with `git clone` covered there.
```

Comment:
You are right, this does require the Move files used in the tutorial available locally. For now I guess it means they have to clone the repo.
But that doesn't sound ideal.
I can see two possible solutions:
1. Put the tutorial and files in a separate repository, which they can clone separately
2. I wonder if there is a way to also download these tutorial files when using git install

Reply:
I think they should sync the repo. I am proposing in Slack we simply make download/clone an optional section in Install. We already link to the Contribute page from there for the step. But that's a circuitous journey for our users.

Reply:
Taking care of this in:
https://github.com/MystenLabs/sui/pull/1374
https://github.com/MystenLabs/sui/pull/1375